### ISSUE TITLE
New Rank system

### ISSUE THREAD

**Issue by Solo0leveling** [ValveSoftware/Dota2-Gameplay#8285]
### Description

 Valve calibration error after update on my account. This is some kind of mistake, since I have my rating in the battle cup 7, 4600- 5500 mmr, and I am thrown to 1.5 to mmr players.
 

Thank you in advance for your response :)

![image](https://user-images.githubusercontent.com/93921738/233756999-8cb0415a-083e-4e6b-9a22-4769228f6aa6.png)

### Example Match ID (and possibly Timestamp)

7117750895

**Comment by Solo0leveling** [comment]
![image](https://user-images.githubusercontent.com/93921738/233756501-666d0bb7-7e9b-46f9-bb81-4ac13d7383b8.png)
Valve calibration error after update on my account. This is some kind of mistake, since I have my rating in the battle cup 7, that is, I can play on divine, and I am thrown to 1.5 to mmr players.

**Comment by commentator24** [comment]
vs divine first game then vs guardian the next game does NOT mean your MMR is divine then suddenly drop to guardian.
You just get more or less mmr vs higher bracket.
i was crusader 4, started 26%, i won first game, lost 2, i get +550 mmr.
but probably because I was vs ancients in my first game.
![image](https://user-images.githubusercontent.com/109324155/233756525-db594442-f294-44d9-b9ca-48b884a0b1f6.png)
as shown here other medals might be different: archon (i was crusader 4), legend and ancient. average doesn't matter.
these ancient players would also ask why vs archons and crusaders?
so if you can't win in 1500 bracket, then it's a big problem. you will lose a lot of mmr.

After your first loss, i think it's about 40-50 mmr per game. i already calibrated and i still get +40 mmr per game after calibration.
You are only matched against 1 player on the opposite team. e.g. yellow vs green. or maybe mid vs mid.
the closer the % to 30% then you wont get as much mmr. 
i'm guessing starting 0% = bigger MMR swing maybe 300-600 per game. if you start 26% then you just need probably 3 games like me and get your mmr back.
assuming 4% is about 500 mmr swing, then if you start at 10%, then 20% would swing you around 2500 mmr.
if you keep losing calibration then you can lose 500-3000 mmr easily.
one of my friend was guardian 5 (i think 1400?), started 11%, won first 5 games, lost 1. calibrated legend 1 (i think 3100). so thats about 1500-1700 mmr.
you really need to win your FIRST calibration game until first loss.
once these divine or ancient player calibrated, there are less divine and ancient players in the player pool to match with, so you gain LESS mmr (because no other previously ancient / divine / immortal are calibrating at the same time) because glicko rating depends on many factors like the opponents, your performance etc. if you want to get back your mmr you probably should calibrate now or just keep complaining that new system is unfair.
i think it's good.

**Comment by Solo0leveling** [comment]
The main problem is that I am a 4-5 position player and my main contribution to the game is to be a team leader who monitors all the moments and indicates what to do. But my first games, my team didn't listen to me at all and I didn't understand the new calibration system itself

**Comment by Solo0leveling** [comment]
So the only option is to play 1-2 positions?

**Comment by commentator24** [comment]
no. i think the only option is to WIN your FIRST game. the rest of games after first loss is about 40 mmr i think. 
if you already had your first calibration game would be nice to screenshot your current % and your results so i can add it to my discord note.

another friend i have, started from 1050 mmr. he is now 2223 mmr with win lose win win. He only plays support as shown.
![image](https://user-images.githubusercontent.com/109324155/233759077-a468553c-43d8-4604-bbdd-5dfa796387c7.png)

**Comment by Solo0leveling** [comment]
Oh, I have 19% cause 5 lost and 8 win. But  after the first game which one was lost, I had 1%

**Comment by Solo0leveling** [comment]
You can write me your discord and I can send you a message in 7 hours. Since I want to get some sleep ;)
